.github/workflows/website-validation.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ubuntu-latest
     name: Build site
     steps:
-      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # ratchet:actions/checkout@v4
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # ratchet:actions/checkout@v4
```

Comment:
I think linking the specific release would be a good idea page, since it allows easily confirming things like that hash not being from a third party fork: https://github.com/actions/checkout/releases/tag/v4.1.7